Core/Base/ConsumerModule.py

```diff
@@ -0,0 +1,81 @@
+"""
+  DIRAC class which is the abstract class for the consumers
+
+  All DIRAC consumers must inherit from the basic class ConsumerModule
+  and override its methods
+"""
+
+from DIRAC.WorkloadManagementSystem.Consumer.ConsumerTools import getConsumerOption
+from DIRAC.WorkloadManagementSystem.Consumer.ConsumerTools import getConsumerSection
```

Comment:
What I mean is that getConsumerOption and getConsumerSection should better be put in a part of the code that is not "WorkloadManagement". Core is possible.